Don't setValueAtTime in volume effect

adroitwhiz opened the pull request:
### Resolves

Resolves #111, resolves #116, and resolves #117 

### Proposed changes

This PR removes the VolumeEffect's setValueAtTime call on its gain node when its volume is set.

### Reason for changes

For bizarre reasons which I believe relate to automation ("ramp to value at time", etc.), the `GainNode`'s 
gain `value` always == 1 when we read it. This resulted in the volume ramping down from 100% whenever it was set, causing a spike in loudness.

adroitwhiz committed:
Don't setValueAtTime in volume effect

adroitwhiz commented:
oh noooo i'll fix the test stuff in the morning

adroitwhiz commented:
legend says the sun hasn't risen to this day

thisandagain commented:
@paulkaplan @ericrosenbaum Would you two mind reviewing this together?

ericrosenbaum committed:
Remove extra wait time in set volume

ericrosenbaum commented:
Hey @adroitwhiz sorry about the delay on this, and thanks for your investigation! @paulkaplan and I are working on it now and making some updates (someday, the sun shall rise).

paulkaplan committed:
Update stop duration and tests

Remove test for ramping because the test audio api doesnt ramp correctly without prior sets

ericrosenbaum commented:
🌅🌅🌅!

ericrosenbaum reviewed (approved):
LGTM! thanks @adroitwhiz and @paulkaplan
